src/components/OnboardingDialog/index.js

```diff
@@ -0,0 +1,156 @@
+import React, { useState } from 'react'
+import styled from '@emotion/styled'
+import PropTypes from 'prop-types'
+import {
+  radius,
+  shadow,
+  space,
+  duration,
+  transition,
+  color,
+  device,
+} from '../../theme'
+import { useTransition } from '../../hooks'
+import { EXITED, ENTERING, ENTERED, EXITING } from '../../hooks/useTransition'
+
+const BodyWrapper = styled.div`
+  background-color: white;
+  border-radius: ${radius.lg};
+  box-shadow: ${shadow.strong};
+  padding: ${space[32]};
+`
+
+const OnboardingDialogBody = ({ children }) => (
```

Comment:
I would add `...props` here as well, since this is exported and thus needs styled components support.